packages/combobox/src/Combobox.ts

```diff
@@ -416,8 +434,14 @@ export class Combobox extends Textfield {
                 ?readonly=${this.readonly}
             />
             ${when(
-                this.pending && !this.disabled && !this.readonly,
-                this.renderLoader
+                this.pendingStateController.isPending() &&
+                    !this.disabled &&
+                    !this.readonly,
+                () => {
+                    return this.pendingStateController.renderPendingState(
+                        this.pendingLabel
+                    );
```

Comment:
`pendingLabel` is a public property on all the consuming components of this controller, we don't need to send a function argument in every call as the controller has direct access to host's public properties.

Reply:
Makes sense!
Thanks for pointing it out